projects/github-actions/repo-gardening/src/utils/labels/get-all-labels.js

```diff
@@ -0,0 +1,43 @@
+/* global GitHub */
+const debug = require( '../debug' );
+
+// Cache for getLabels.
+const cache = {};
+
+/**
+ * Get all the labels available in the repo.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} filter  - Optionally filter to only return a subset of labels. Use a regex pattern.
+ * @return {Promise<Array>} Promise resolving to an array of all labels in the repo.
+ */
+async function getAllLabels( octokit, owner, repo, filter = '' ) {
```

Comment:
I wonder whether "getAvailableLabels" might be more accurately descriptive.

Reply:
I like it. f95209287d7735dfd08b4a149e836ba8e6f6c367